src/AISmart.Application.Contracts/GEvents/Social/SocialResponseEvent.cs

```diff
@@ -12,4 +12,5 @@ public class SocialResponseEvent:EventBase
     [Id(1)]  public string ChatId { get; set; }
     [Description("Optional ID of the message to which this message is a reply.")]
     [Id(2)]   public string? ReplyMessageId { get; set; }
+    [Id(3)]   public string? TweetId { get; set; }
```

Comment:
This should not be needed. Anyone receiving social event should not need to know about the tweet id